src/main/java/com/learning/mfscreener/service/PortfolioService.java

```diff
@@ -115,17 +122,10 @@ public String upload(MultipartFile multipartFile) throws IOException {
 
     UploadResponseHolder findDelta(String email, String name, CasDTO casDTO) {
```

Comment:
@coderabbitai, suggest code changes without breaking functionality to reduce cognitive complexity.